board/aarch64/raspberrypi-rpi64/README.md

```diff
@@ -1,59 +1,303 @@
+admin@infix:/config/container/doom/> set image docker://mattiaswal/alpine-doom:latest
+admin@infix:/config/container/doom/> set privileged
+admin@infix:/config/container/doom/> edit mount udev
+admin@infix:/config/container/…/udev/> set type bind
+admin@infix:/config/container/…/udev/> set target /run/udev/
+admin@infix:/config/container/…/udev/> set source /run/udev/
+admin@infix:/config/container/…/udev/> end
+admin@infix:/config/container/doom/> edit mount xorg.conf
+admin@infix:/config/container/…/xorg.conf/> set content U2VjdGlvbiAiT3V0cHV0Q2xhc3MiCiAgSWRlbnRpZmllciAidmM0IgogIE1hdGNoRHJpdmVyICJ2YzQiCiAgRHJpdmVyICJtb2Rlc2V0dGluZyIKICBPcHRpb24gIlByaW1hcnlHUFUiICJ0cnVlIgpFbmRTZWN0aW9uCg==
+admin@infix:/config/container/…/xorg.conf/> set target /etc/X11/xorg.conf
+admin@infix:/config/container/…/xorg.conf/> end
+admin@infix:/config/container/doom/> edit volume var
+admin@infix:/config/container/…/var/> set target /var
+admin@infix:/config/container/…/var/> leave
 ```
 
 > [!NOTE]
-> The `xorg.conf` [content mount][2] is a nifty detail of Infix that
-> allows you to keep all the relevant configuration in a single file.
-> The deta is "simply" `base64` encoded, so you do not really need the
-> features of the Infix CLI, everything can be set up remotely [using
-> `curl`][3] if you like.
+> The `xorg.conf` [content mount][3] is a useful Infix feature that allows you
+> to keep all configuration in a single file. The data is base64 encoded, and
+> everything can be set up remotely [using `curl`][4] if preferred.
+
+### Power Supply Requirements
+
+Proper power supply is critical for stable operation:
+
+- **Raspberry Pi 4B:** 5V/3A USB-C power supply (official recommended)
+- **Raspberry Pi 3B:** 5V/2.5A micro-USB power supply
+- **Compute Module 4:** Power requirements depend on carrier board
+
+Inadequate power can cause:
+- Random reboots or crashes
+- USB device failures
+- Network disconnections
+- Corrupted storage
+
+<img align="right" src="gpio-pinout.png" alt="GPIO Pinout" width=300 padding=10>
+
+### Serial Console Access (Optional)
+
+A serial console is useful for debugging but not required for Pi 3B/4B, since
+the factory configuration enables network access via DHCP. For CM4 and other
+variants without factory configuration, serial console access is required for
+initial setup.
+
+To connect via serial:
+
+1. Connect a USB-to-TTL serial adapter (3.3V) to GPIO pins:
+   - GND → Pin 6 (Ground)
+   - TX → Pin 8 (GPIO 14, RXD)
+   - RX → Pin 10 (GPIO 15, TXD)
+2. Use 115200 baud, 8 data bits, no parity, 1 stop bit (115200 8N1)
+3. Connect using `screen`, `minicom`, or similar terminal emulator
+
+The image shows the standard 40-pin GPIO header pinout. For serial console:
+- **Pin 6** (black) = Ground
+- **Pin 8** (orange) = TxD (UART) - connect to RX on your adapter
+- **Pin 10** (orange) = RxD (UART) - connect to TX on your adapter
+
+> [!WARNING]
+> Use only 3.3V serial adapters. 5V adapters will damage your Raspberry Pi!
```

Comment:
Also, maybe note what to do with the red connector on a USB-to-TTL cable

Reply:
Thanks, will do!